packages/edit-site/src/components/header/theme-exporter/index.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * External dependencies
+ */
+import { saveAs } from 'file-saver';
+
+/**
+ * WordPress dependencies
+ */
+import { useSelect } from '@wordpress/data';
+import { useCallback } from '@wordpress/element';
+import { MenuItem } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import getBaseThemeZip from './get-base-theme-zip';
+
+export default function ThemeExporter( { ids, templatePartIds } ) {
```

Comment:
We should probably allow this to be disabled somehow and account for permissions.

Reply:
>disabled somehow

Through the editor settings, bootstrapped on the server? So that it can be both filtered and toggled with a store action?

>account for permissions

Shouldn't anyone who can edit templates also be able to export them?